packages/truffle-debugger/lib/controller/sagas/index.js

```diff
@@ -108,6 +113,9 @@ function* stepNext() {
  * step.
  */
 function* stepInto() {
+  if (!(yield select(controller.current.loaded))) {
+    return;
+  }
```

Comment:
There's a lot of duplication here. Did you have any thoughts about how we might abstract this? Just curious.

Reply:
Hm... we could move the check to the listener saga.  That wouldn't affect internal calls, but, we shouldn't be calling these internally in the first place if there's no transaction loaded!  How's that sound?